code/src/components/subintro/subintro.scss

```diff
@@ -0,0 +1,23 @@
+@import "@design-system";
+
+.spa-subintro {
+    display: flex;
+    align-items: center;
+    gap: spacing(25);
+
+    &_element {
+        display: flex;
+        gap: spacing(7);
+        padding: spacing(10) 0px spacing(10);
```

Comment:
Maybe little fix, but maybe use just `0` instead of `0px`

Reply:
Done